gems/manageiq-providers-amazon/spec/models/manageiq/providers/amazon/cloud_manager/orchestration_stack_spec.rb

```diff
@@ -100,6 +100,28 @@
         end
       end
     end
+
+    describe '.fix_options(options)' do
+      let(:v2_options) do
```

Comment:
@bzwei I think we should put the `v2_options` together with the v2 test below, as is done for `v1_options`.